packages/mirta-rollup/README.ru.md

```
@@ -3,26 +3,174 @@
+
+# При сборке инструментов командной строки без exports
+rollup -c node:@mirta/rollup/config-package --config-skip-exports
+
 ```
-## Использование
+#### Особенности
+
+- Вход: `src/index.ts`;
+- Формат: `es` → `.mjs`;
+- Валидация:
+    - проверка соответствия `src/` ↔ `package.json#exports`;
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)